File: Bars.GisIntegration.UI/libs/B4/model/GisDictionary.js
Ext.define('B4.model.GisDictionary', {
    extend: 'B4.base.Model',
    idgen: {
        type: 'sequential',
        prefix: 'ID_'
    },
    fields: [
        { name: 'RegistryNumber', type: 'int' },
        { name: 'Name' },
        { name: 'Group' },
        { name: 'Modified' }
    ],
    proxy: {
        type: 'memory',
        reader: {
            type: 'json'
        }
    }
});

File: Bars.GisIntegration.Base/Dictionaries/Impl/GisDictionary.cs
namespace Bars.GisIntegration.Base.Dictionaries.Impl
{
    using System;

    using Bars.GisIntegration.Base.Enums;

    /// <summary>
    /// Справочник ГИС ЖКХ
    /// </summary>
    public class GisDictionary
    {
        /// <summary>
        /// Реестровый номер
        /// </summary>
        public string RegistryNumber { get; set; }

        /// <summary>
        /// Наименование
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// Группа справочника
        /// </summary>
        public DictionaryGroup Group { get; set; }

        /// <summary>
        /// Дата последнего изменения
        /// </summary>
        public DateTime Modified { get; set; }
    }
}